packages/spectral/src/generators/componentManifest/cli.ts

```diff
@@ -122,8 +124,8 @@ export const runMain = async (process: NodeJS.Process) => {
   await createComponentManifest({
     component,
     dryRun: flags.dry_run.value,
-    includeSignature: flags.include_signature.value,
-    packageName: flags.name.value ?? `@components/${component.key}-manifest`,
+    validateSignature: flags.validate_signature.value,
+    packageName: flags.name.value ?? `@component-manifests/${component.key}`,
```

Comment:
i _think_ we'll need to update the Readme as well

Reply:
Nope, this `packageName` is passed down to the readme.

![image](https://github.com/prismatic-io/spectral/assets/5714439/4f1d32f3-43c2-42a8-bfc8-111bec0857f7)